src/WidenImage.css

```diff
@@ -0,0 +1,61 @@
+
+.wideImageOverride {
+    /* Center images. */
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+@media (orientation: portrait) {
+    .wideImageOverride {
+        width:100% !important;
+        max-width:100% !important;
+        height:auto !important;
+    }
+
+    /* Tablet override */
+    @media (min-device-width: 768px) {
```

Comment:
I don't mind if we _pun_t on this for now but if we do, would you mind adding our definition of wide screen width to the readme?

FWIW, it looks like CSS does actually have variable support though:

- https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_variables
- http://caniuse.com/css-variables/embed

I've unknowingly used these in the tiny bit of Polymer I tried but I thought it was like a hack specific to web components :D